src/frame_main.cpp

```diff
@@ -31,17 +31,17 @@
 int main(int argc, char const* argv[])
 {
     using namespace miral;
-    MirRunner runner{argc, argv};
+    MirRunner* const runner = new MirRunner(argc, argv);
```

Comment:
Do not create objects with raw `new` - they are not "owned" and will leak

```suggestion
    MirRunner runner{argc, argv};
```